.gitattributes

```diff
@@ -4,3 +4,14 @@
 
 styles/* linguist-vendored
 test/**/*.txt linguist-vendored
+
+/.editorconfig        export-ignore
+/.gitattributes       export-ignore
+/.github              export-ignore
+/.gitignore           export-ignore
+/.php_cs.dist         export-ignore
+/.travis              export-ignore
```

Comment:
Will this cover both the `.travis` folder and `.travis.yml` file?

Reply:
@allejo 
I am not sure about it, but we can add a separate line for `.travis.yml`

Reply:
Yea if you could please have it include the folder and file, that'd be awesome!